pkg/controller/kogitoapp/resource/deployment_config.go

```diff
@@ -66,19 +67,25 @@ var (
 		For Spring: https://github.com/infinispan/infinispan-spring-boot/blob/master/infinispan-spring-boot-starter-remote/src/test/resources/test-application.properties
 	*/
 
-	envVarInfinispanQuarkus = map[string]string{
-		envVarInfinispanServerList:    "QUARKUS_INFINISPAN_CLIENT_SERVER_LIST",
-		envVarInfinispanUseAuth:       "QUARKUS_INFINISPAN_CLIENT_USE_AUTH",
-		envVarInfinispanUser:          "QUARKUS_INFINISPAN_CLIENT_AUTH_USERNAME",
-		envVarInfinispanPassword:      "QUARKUS_INFINISPAN_CLIENT_AUTH_PASSWORD",
-		envVarInfinispanSaslMechanism: "QUARKUS_INFINISPAN_CLIENT_SASL_MECHANISM",
+	propertiesInfinispanQuarkus = map[string]string{
+		appPropInfinispanServerList:    "quarkus.infinispan-client.server-list",
+		appPropInfinispanUseAuth:       "quarkus.infinispan-client.use-auth",
+		appPropInfinispanSaslMechanism: "quarkus.infinispan-client.sasl-mechanism",
+
+		appPropKafkaBootstrapServerList: "quarkus.kafka.bootstrap-servers",
```

Comment:
I think that `bootstrap.servers` should be used for both Quarkus and SpringBoot.

Reply:
https://kafka.apache.org/documentation/#bootstrap.servers

Reply:
https://quarkus.io/guides/kafka-streams#topic-configuration